src/core/toolbox/providers/Toolbox.js

```diff
@@ -79,7 +104,7 @@ export class Toolbox {
     }
 
     onMouseUp (event) {
-        if (!this.activeTool) return;
+        if (!this.activeTool || !this.mouseDown) return;
```

Comment:
can this lead in some misbehavior ?

Reply:
Hmm, not sure. I haven't encoutered anything yet. Can you think of a case where you would want to react to a mouse up when there was no previous mouse down registered (or it was canceled)?